File: cpp/example/src/BubbleSort/bubbleSort.cpp
#include "bubbleSort.h"

vector<int> bubbleSort::naive(vector<int>& nums){
    for (int i=0;i<nums.size();i++){
        for (int j=i+1;j<nums.size();j++){
            if (nums[i]>nums[j]){
                int tmp = nums[i];
                nums[i] = nums[j];
                nums[j] = tmp;
            }
        }
    }
    return nums;
}

File: cs/sample/Sample/BubbleSort/BubbleSort.cs
using System;
using System.Collections.Generic;

namespace Sample.BubbleSort{
    public class bubbleSort{
        public static int[] naive(int[] nums){
            for (int i=0;i<nums.Length;i++){
                for (int j=i+1;j<nums.Length;j++){
                    if (nums[i]>nums[j]){
                        (nums[i],nums[j])=(nums[j],nums[i]);
                    }
                }
            }
            return nums;
        }
    }
}
